src/main/groovy/life/qbic/datamodel/accounting/Product.groovy

```diff
@@ -1,9 +1,11 @@
 package life.qbic.datamodel.accounting
```

Comment:
isnt Product etc not also a DTO?

Reply:
Ah its the old and deprecated Product class, is it?

Reply:
Yes :)